ebu_tt_live/node/delay.py

```diff
@@ -76,6 +83,29 @@ def update_children_timing(element, timebase, delay_int):
                 update_children_timing(child.value, timebase, delay_int)
 
 
+def update_body_timing(body, timebase, delay_int):
+
```

Comment:
1. I don't 100% agree. I agree to say that we always _insert_ a body begin time, but sometimes, if the end attribute is already present, we _update_ it.

2. I struggled a bit with how to formulate this one. In fact, we could have a body begin time, but if it's the case, it must be None. This is how I did the assertion in 2d2ffb1.